src/app/static/src/app/components/projects/ShareProject.vue

```diff
@@ -150,6 +160,20 @@
                     }
                 });
             },
+            cycleInputs(email, index){
+                    const lastInputIndex = this.emailsToShareWith.length - 1
+                    const lastInput = this.$el.querySelectorAll("#shareInputs > div > input")[lastInputIndex]! as HTMLElement
+                    const currentInput = this.$el.querySelectorAll("#shareInputs > div > input")[index]! as HTMLElement
+                    const okBtn = this.$refs.okBtn as HTMLElement
+
+                if (index === lastInputIndex && !email.value){
+                    currentInput.blur()
+                    okBtn.focus()
+                } else {
+                    currentInput.blur()
+                    lastInput.focus()
```

Comment:
Should this not focus index + 1?

Reply:
I made the design choice to move to the last (empty) input rather than the next after validation as I figured this is where the user would want to make future input if any.